install/kubernetes/helm/istio/charts/gateways/values.yaml

```diff
@@ -253,8 +253,11 @@ istio-ilbgateway:
     #  cpu: 1800m
     #  memory: 256Mi
   loadBalancerIP: ""
-  serviceAnnotations:
-    cloud.google.com/load-balancer-type: "internal"
+  serviceAnnotations: {}
+  #  cloud.google.com/load-balancer-type: "internal"
```

Comment:
there is a small problem with this change.  no `serviceAnnotations` annotation is defined by default.

Reply:
+1, adding the comments is ok with me as long as we keep the default